.github/actions/bitcoin-int-tests/Dockerfile.bitcoin-tests

```diff
@@ -21,6 +21,7 @@ RUN cargo test -- --test-threads 1 --ignored tests::neon_integrations::bitcoind_
 RUN cargo test -- --test-threads 1 --ignored tests::neon_integrations::liquid_ustx_integration
 RUN cargo test -- --test-threads 1 --ignored tests::neon_integrations::stx_transfer_btc_integration_test
 RUN cargo test -- --test-threads 1 --ignored tests::neon_integrations::bitcoind_forking_test
+RUN cargo test -- --test-threads 1 --ignored tests::neon_integrations::should_fix_2771
```

Comment:
I'm not sure how these `bitcoin-tests` work, but my understanding is is that this test does not actually "fail" on failure.. it just kind of hangs.. or is that wrong?

Reply:
When hanging for more for more than N minutes (10 I believe), tests end up aborting and failing.

Reply:
https://github.com/blockstack/stacks-blockchain/blob/3382ecdc73dc456f51fe40b5094d321a7f6e8407/testnet/stacks-node/src/tests/neon_integrations.rs#L273